src/referrals-queue/referrals-queue.resource.ts

```diff
@@ -1,8 +1,13 @@
 import { openmrsObservableFetch } from "@openmrs/esm-api";
 import { map } from "rxjs/operators";
 
-export function getReferrals(): Observable<Object[]> {
+export function getReferrals({
+  fromDate,
+  toDate,
+  patient = null
+}): Observable<Object[]> {
   return openmrsObservableFetch(
-    `/ws/rest/v1/reportingrest/reportdata/cd7dfde7-764a-4da6-81c2-d5887ed1df51?startDate=2020-01-01&endDate=2020-04-01&locale=en`
+    `/ws/rest/v1/reportingrest/reportdata/cd7dfde7-764a-4da6-81c2-d5887ed1df51?startDate=${fromDate}&endDate=${toDate}&locale=en` +
+      (patient ? "&patient=" + patient : "")
```

Comment:
Did you mean to add this in (the patient parameter)?  Looks like it in preparation for server-side filtering and not used?  I guess there's no harm in leaving it around.

Reply:
Oh yeah, that is what happened. I'll clean it up. Thanks!